game-app/game-core/src/main/java/games/strategy/triplea/ui/AdditionalTerritoryDetails.java

```diff
@@ -0,0 +1,26 @@
+package games.strategy.triplea.ui;
+
+import games.strategy.engine.data.Territory;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class AdditionalTerritoryDetails {
+  private final List<Function<Territory, String>> additionalTerritoryDetailFunctions =
```

Comment:
Since this is already a member of AdditionalTerritoryDetails, I don't think we need to spell out the full class name here again. How about just `detailProviders` and  renaming the two methods to add/remove them accordingly?

Reply:
Sounds good